examples/using-graphql-typegen/.eslintrc.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+  root: true,
+  parser: "@typescript-eslint/parser",
+  plugins: [
+    "@typescript-eslint"
+  ],
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  overrides: [
+    {
+      files: ['*.ts', '*.tsx'],
+      processor: '@graphql-eslint/graphql',
+      extends: ['eslint:recommended'],
+      env: {
+        es6: true,
+      },
+    },
```

Comment:
It's very important that you don't have any rules configured at the top-level config, and to move all configurations into the `overrides` section. Since JavaScript rules can't run on GraphQL files and vice versa, if you have rules configured at the top level, they will try to also execute for all overrides, as ESLint's configs cascade.

```suggestion
  overrides: [
    {
      files: ['*.ts', '*.tsx'],
      processor: '@graphql-eslint/graphql',
      parser: "@typescript-eslint/parser",
      extends: [
        "eslint:recommended",
        "plugin:@typescript-eslint/recommended"
      ],
      env: {
        es6: true,
      },
    },
```